egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -59,6 +59,24 @@ def ParseSpliceString(splice_indexes, label_delay=None):
             'num_hidden_layers':len(splice_array)
             }
 
+def ParseLstmDelayString(lstm_delay):
+    ## Work out lstm_delay e.g. lstm_delay = [ [-1], [-1, 1], [-2] ]
```

Comment:
The lstm_delay format in the comment does not match your input. Make them consistent.